cmd/package-server/main.go

```diff
@@ -19,6 +20,11 @@ func main() {
 	options := server.NewPackageServerOptions(os.Stdout, os.Stderr)
 	cmd := server.NewCommandStartPackageServer(ctx, options)
 	cmd.Flags().AddGoFlagSet(flag.CommandLine)
+	// klog flags used by dependencies need to be explicitly initialized now
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+	klog.InitFlags(klogFlags)
+	cmd.Flags().AddGoFlagSet(klogFlags)
```

Comment:
You've been closer to the ground than I have on this - any idea on where I would be able to learn more about this further, or could provide some more color on the comment you added here? It's possible I'm missing something obvious and I need some more caffeine in me.

Reply:
See: https://pkg.go.dev/k8s.io/klog#readme-how-to-use-klog
"Use klog.InitFlags(nil) explicitly for initializing global flags as we no longer use init() method to register the flags"

Reply:
Interesting - looking at the 1.23 release notes, it looks like the majority of the klog-related flags were deprecated outside of the `-v` flag. Did you need to add this as compilations were failing, or because the packageserver was crashing at runtime because the new changes resulted in mis-configuration?

Reply:
Just pulled these changes down locally and deployed w/o these klog changes and it results in runtime errors over time (vs. static compilation failures).